modules/config.py

```
@@ -741,23 +741,36 @@ def import_configurations(self):
+                    print(f"WARNING: Could not migrate legacy config at {legacy_path}: {e}", file=sys.stderr)
         print(f"DEBUG: Checking local config path FULL PATH: {local_path.resolve()}", file=sys.stderr)
         print(f"DEBUG: Current working directory: {Path.cwd()}", file=sys.stderr)
         try:
-            # Check if we can write to the current directory
+            local_dir = local_path.parent
+            local_dir.mkdir(parents=True, exist_ok=True)
+
             # Check if a config file exists and is empty (possibly from a failed previous run)
             # If empty, we can safely delete it and use the local path.
             if local_path.exists() and local_path.stat().st_size == 0:
                  try: local_path.unlink()
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)